tests/integration/llm/client.py

```diff
@@ -397,6 +397,24 @@ def get_model_name():
     }
 }
 
+ds_smoothquant_model_spec = {
+    "gpt-j-6b": {
+        "max_memory_per_gpu": [9.0, 10.0, 11.0, 12.0],
+        "batch_size": [1, 2, 4, 8],
+        "seq_length": [64, 128, 256],
+    },
+    "llama2-13b": {
+        "max_memory_per_gpu": [22.0],
+        "batch_size": [1],
```

Comment:
Lets do a test with batch size 2+ as well, we don't need the whole series but we will be able to observe output errors if batch is not handled correctly.

Reply:
Will update. Need to update the `max_memory_per_gpu` with some more accurate numbers